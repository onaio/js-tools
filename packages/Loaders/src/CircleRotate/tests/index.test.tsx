import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';
import React from 'react';

import { CircleRotate } from '..';

describe('Loaders.CircleRotate', () => {
  it('renders without crashing', () => {
    const wrapper = mount(<CircleRotate />);

    expect(toJson(wrapper.find('CircleRotate'))).toMatchSnapshot();
  });
});
