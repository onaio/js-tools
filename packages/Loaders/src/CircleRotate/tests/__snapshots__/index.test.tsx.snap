// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loaders.CircleRotate renders without crashing 1`] = `
.c0 {
  display: inline-block;
  -webkit-transform: translateZ(1px);
  -ms-transform: translateZ(1px);
  transform: translateZ(1px);
}

.c0 div {
  display: inline-block;
  width: 64px;
  height: 64px;
  margin: 8px;
  border-radius: 50%;
  background: tomato;
  -webkit-animation: cGjRkh 1.2s cubic-bezier(0,0.2,0.8,1) infinite;
  animation: cGjRkh 1.2s cubic-bezier(0,0.2,0.8,1) infinite;
}

<CircleRotate
  animationSpeed={2.4}
  color="tomato"
  scaleSizeBy={1}
  scaleSpeedBy={1}
>
  <styled.div
    animationSpeed={2.4}
    color="tomato"
    scaleSizeBy={1}
    scaleSpeedBy={1}
  >
    <div
      className="c0"
      color="tomato"
    >
      <div />
    </div>
  </styled.div>
</CircleRotate>
`;
